deephaven_ipywidgets/deephaven.py

```diff
@@ -7,15 +7,17 @@
 """
 Module for displaying Deephaven widgets within interactive python environments.
 """
+from __future__ import annotations
```

Comment:
Unused import?

Reply:
This is so I can reference DeephavenWidget in the type of _cleanup while insuring compatibility